generate/generate.go

```diff
@@ -832,6 +832,15 @@ func (s *service) GenerateCode() ([]byte, error) {
 		pn("	return json.Unmarshal(resp, result)")
 		pn("}")
 	}
+	if s.name == "ResourcetagsService" {
+		pn("func getSortedKeysFromMap(m map[string]string) (keys []string) {")
+		pn("	for k := range m {")
+		pn("		keys = append(keys, k)")
+		pn("	}")
+		pn("	sort.Strings(keys)")
+		pn("	return")
+		pn("}")
+	}
```

Comment:
I think this function should be in the `cloudstack.go` file. So somewhere around line 505 of this file...

Reply:
Nice catch @svanharmelen. I really chose to put that helper func there arbitrarily. Sorry :smile: I'm moving it to the general code.